tools-and-communication/creating-videos.instructor.md

```diff
@@ -0,0 +1,76 @@
+  - You can work errors into your lessons and ask students to take a moment to solve the problem, and then make it into a mini-lesson about how to debug or research problems.
+  - Mistakes also help humanize you to students
+- **Give practice**
+  - Mistakes above can serve this purpose, but you can give students a question or problem and ask them to pause the video and figure it out, and then work through how you resolve things.
+  - You can also use the Learn Challenges in the curriculum.
+- **Be Aware of where the thumbnail will be**
+  - Both Panopto and Zoom put thumbnails of the speaker in a fixed location in the video.  *Don't put code where it will be covered up!*
+  - Camtasia does allow you to shift the location of the thumbnail in editing.
+
+## Workflow
+
+To help adding video lessons the flow goes like this:
+
+1.  Find the [Video content Airtable](https://airtable.com/tblXx1WLkPwJqp46T)
+1.  Identify a video assigned to you, or pick one which is unclaimed
+1.  Review the lesson, make suggestions/edits for the curriculum team.
+1.  Mark the video as "In Progress" in the airtable
+1.  Record the lesson making edits as needed
+1.  Upload the video lessons to [panopto](https://adaacademy.hosted.panopto.com/Panopto/Pages/Sessions/List.aspx#folderID=%22bf8fa6c1-e3d3-46b2-b3f9-accb01698ec1%22)
+1.  Create a playlist (even for a 1 video lesson) and attach videos to it.
+1.  Place the embedded iframe for the playlist into the learn lesson.
+1.  Git-add-commit and learn publish the lesson
+1.  Watch at least a bit of each video, just to make SURE you didn't make a mistake 😊
+1.  Mark the lesson as done in the airtable.
+
+## Questions To Answer Over Time
+
+These are questions I have about best-practices that we'll have to answer over time.
+
+- Do I need to have my face/thumbnail on videos?
+  - I (Chris) am unconvinced that the thumbnail of our lovely faces is helpful for learning.  Personally I think it distracts from the code we want them to follow and concepts we want students to learn.  That said, it helps students maintain a human connection to their instructors and several students (C13) have asked to keep them.
+- Do we need more production value (intro screens, editing etc)?
+  -  Would higher production values give us any "bang for our buck?"  Currently I (Chris) do not think so.  Especially as we're not selling the videos, it's probably not worthwhile.
+- Should we invest in alternative software?
+  - If Panopto shifts to a different liscense or goes under, Youtube or other streaming services could be used.  We should look into a backup storage service.  I (Chris) am storing mine on a used NAS I bought for myself as a backup.
+  - [Camtasia](https://www.techsmith.com/camtasia-upgrade.html) is a good and widely used software package to do screencasting in education.  It is pretty easy to add some production value to a video and add neat effects. 
```

Comment:
Love! I wonder if it'd be worth considering buying this for all Instructors...that's definitely a convo outside of this PR though